app/src/main/res/values/group_setting_style.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="groups_number_setting">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:imeOptions">actionDone</item>
+        <item name="android:singleLine">true</item>
+        <item name="android:gravity">center</item>
+        <item name="android:inputType">number</item>
+        <item name="android:layout_marginLeft">5dp</item>
```

Comment:
Can we try to use multiples of 4 for all the spacing? So 5 -> 4, 3 -> 4, 10 -> 8 or 12.